beta/docker/README.md

```diff
@@ -0,0 +1,231 @@
+# [Beta] Deploy 1Password SCIM Bridge using Docker
+
+This example describes how to deploy 1Password SCIM Bridge as a stack using [Docker swarm](https://docs.docker.com/engine/swarm/key-concepts/#what-is-a-swarm). The stack includes two services (one each for the SCIM bridge container and the container for the required Redis cache), a Docker secret for the `scimsession` credentials, a Docker config for Redis, and optional secrets for customers conecting to Google Workspace.
+
+## In this folder
+
+- [`README.md`](./README.md): the document that you are reading. 👋😃
+- [`compose.template.yaml`](./compose.template.yaml): a [Compose specification](https://docs.docker.com/compose/compose-file/) format  [Compose file](https://docs.docker.com/compose/compose-file/03-compose-file/) for 1Password SCIM Bridge.
+- [`compose.gw.yaml`](./compose.gw.yaml): an [override configuration](https://docs.docker.com/compose/multiple-compose-files/merge/) to merge the configuration necessary for customers integrating with Google Workspace.
+- [`redis.conf`](./redis.conf): a Redis confguration file to load the Redis cache with the base configuration required for SCIM bridge
+- [`scim.env`](./scim.env): an environment file used to configure SCIM bridge
+
+## Overview
+
+The open source Docker Engine tooling can be used to deploy 1Password SCIM Bridge on any supported Linux distribution. A single-node swarm is sufficient. This example assumes that the Linux server is exposed directly to the public internet with Docker acting as a reverse proxy to the SCIM bridge container. Other configurations are possible, but not documented in this deployment example.
+
+## Prerequisites
+
+- Linux VM or bare metal server supporting Docker
+- Docker Engine (v20.10.3+) (see [Docker Engine installation overview](https://docs.docker.com/engine/install/#server))
+- public DNS record pointing to the Linux server
+
+## Get started
+
+1. Create a public DNS record that points to the public IP address of the Linux server for your SCIM bridge. For example, `scim.example.com`.
+
+2. [Install Docker Engine](https://docs.docker.com/engine/install/#server) on the server (follow the Server instructions; Docker Desktop is not needed). We also recommend following the [post-install steps](https://docs.docker.com/engine/install/linux-postinstall/) as noted in the documentation to enable running Docker as a non-root user and ensure that Docker Engine starts when the Linux system boots.
+
+3. All following steps can be run directly on the Linux server, or by connecting remotely using the `--host` parameter or `docker context`. If you are already using SSH to connect to the Linux server, you can create and use a Docker context on your local computer that is connected to your 1Password account (these commands require Docker Engine or Docker Desktop to be installed on your computer).
```

Comment:
I wonder if we should be as simple and straightforward as possible here. Maybe we should just lean into the "directly on the Linux server" and if someone is experienced enough and knows how to do this other ways, they can go ahead and do so. 

I'm not exactly sure that is meant by
>connected to your 1Password account

Just a couple thoughts, no demands here.

Reply:
My thoughts here are that someone needs to get the `scimsession` credentials and transfer them to the server where they are deploying their SCIM bridge, and this provides a sensible means to do so.

In the common case where someone is working from their daily driver workstation that they use to set up provisioning  for their 1Password account and using SSH to connect to the Linux server, this provides an easy means to create Docker secrets from the same workstation, and potentially even load them straight from their 1Password account via `op read` the same way I hint at for the bearer token in the callout. I intended to throw some more similar hints about that.

Creating a Docker context also sets it up nicely for the same admin to manage it in the future or as reference for someone else to create to do the same.